hbase/stackable/patches/apply_patches.sh

```diff
@@ -4,14 +4,21 @@
 set -eu
 set -o pipefail
 
-# Check if $1 (VERSION) is provided
+# Check if $1 (PRODUCT) is provided
 if [ -z "${1-}" ]; then
-  echo "Please provide a value for VERSION as the first argument."
+  echo "Please provide a value for PRODUCT as the first argument."
   exit 1
 fi
 
-VERSION="$1"
-PATCH_DIR="patches/$VERSION"
+# Check if $2 (VERSION) is provided
+if [ -z "${2-}" ]; then
+  echo "Please provide a value for VERSION as the second argument."
+  exit 1
+fi
+
+PRODUCT="$1"
+VERSION="$2"
+PATCH_DIR="patches/$PRODUCT/$VERSION"
```

Comment:
I see what this is getting at. It would be nice to unify these scripts across all the products at some point, but I think it's fine to defer that to a separate issue.

Reply:
#677